metacatalog/api/catalog.py

```diff
@@ -31,11 +33,16 @@ def get_uuid(session: Session, uuid: str, not_found='raise'):
     if group is not None:
         return group
 
+    # check if a Person exists
+    group = api.find_person(session, uuid=uuid)
+    if group is not None:
+        return group
```

Comment:
While this will work (at least I assume so), it would be way better to rename the local variable `group` to i.e. `person` here for two reasons:
1. `person` describes way better what this variable is about
2. `group` has been used above. It is not considered good practice to re-use a variable in the same scope like this.

Reply:
Sure, I didn´t see that